Refresh PostHog vs Google Analytics (#8919)
PostHog/posthog.com: contents/blog/posthog-vs-ga4.mdx

import { ComparisonHeader } from 'components/ComparisonTable/header'
import { ComparisonRow } from 'components/ComparisonTable/row'

## How is PostHog different?

### 1. We're an all-in-one platform

> PostHog combines [product analytics](/product-analytics) and [web analytics](/web-analytics) with other useful tools, such as [session replay](/session-replay), [A/B testing](/ab-testing), and [surveys](/surveys), into one tightly integrated platform. Everything you need with one vendor, one contract, and for less than buying separately. And you can self-serve, too.

Reviewer: ```suggestion
PostHog combines [product analytics](/product-analytics) and [web analytics](/web-analytics) with other useful tools, such as [session replay](/session-replay), [A/B testing](/ab-testing), and [surveys](/surveys) into one tightly integrated platform. Everything you need from a single vendor with a single contract for less than buying separately. And you can self-serve, too.
```

> PostHog combines [product analytics](/product-analytics) and [web analytics](/web-analytics) with other useful tools, such as [session replay](/session-replay), [A/B testing](/ab-testing), and [surveys](/surveys) into one tightly integrated platform. Everything you need from a single vendor with a single contract for less than buying separately. And you can self-serve, too.

Author: ```suggestion
PostHog combines [product analytics](/product-analytics) and [web analytics](/web-analytics) with other useful tools, such as [session replay](/session-replay), [feature flags](/feature-flags], [A/B testing](/ab-testing), and [surveys](/surveys) into one tightly integrated platform. Everything you need from a single vendor with a single contract for less than buying separately. And you can self-serve, too.
```

### 2. We build for developers first

> This means you get support from the engineers who _actually build the product_. We also have [extensively documented APIs](/docs/api) and a [SQL query builder](/docs/hogql), so you can you can analyze data how you want. Feature flags make it easy to [test in production](/product-engineers/testing-in-production), conduct [phased rollouts](/tutorials/phased-rollout), and so much more.

Reviewer: You could mention data warehouse as a way to query external sources here.

> This means you get support from the engineers who _actually build the product_. We also have [extensively documented APIs](/docs/api) and a [SQL query builder](/docs/hogql), so you can you can analyze data how you want. Feature flags make it easy to [test in production](/product-engineers/testing-in-production), conduct [phased rollouts](/tutorials/phased-rollout), and so much more.

Author: ```suggestion
This means you get support from the engineers who _actually build the product_. We also have [extensively documented APIs](/docs/api) and a [SQL query builder](/docs/hogql), so you can you can analyze data how you want, and you can query external sources by importing them into [our data warehouse](/docs/data-warehouse).
```

  <ComparisonRow column1={true} column2={true} feature="Funnels" description="Visualize conversion rates and drop-offs" />
  <ComparisonRow column1={true} column2={true} feature="User paths" description="Track user flows and where they drop-off" />
  <ComparisonRow column1={true} column2={true} feature="Retention" description="Analyze user and revenue retention" />
  <ComparisonRow column1={true} column2={true} feature="Monetization analytics" description="Track purchase value, LTV, and other revenue metrics" />
  <ComparisonRow column1={false} column2={true} feature="Advertising analytics" description="Track ROI on Google Ads and other marketing campaigns" />
  <ComparisonRow column1={true} column2={true} feature="Real-time view" description="Monitor activity on your site or app as it happens." />
  <ComparisonRow column1={false} column2={true} feature="Predictive insights" description="AI-powered alerts when metrics change" />
</ComparisonTable>

> You can also go deeper on user behavior by utilizing [heatmaps](/docs/toolbar/heatmaps), formulas, and the [custom SQL insights](/docs/hogql).

Reviewer: ```suggestion
You can also go deeper on user behavior by utilizing [heatmaps](/docs/toolbar/heatmaps), scrollmaps, formulas, and the [custom SQL insights](/docs/hogql).
```

</ComparisonTable>

### Integrations

It's hard to import data into Google Analytics because:

1. The data type and format you're allowed to [import](https://support.google.com/analytics/answer/10071301) is restrictive. 
2. You either need to constantly upload CSV files manually, or set up an SFTP server to automatically do this for you.

In contrast, PostHog is built to be your single source of truth, so it's simple to import data from other sources using our [built-in data warehouse](/docs/data-warehouse), or send PostHog data to other tools. You can even [create your own integrations](/docs/cdp/build). 

Below is a comparison of some of the most popular apps – see our [data pipeline](/docs/cdp/airbyte-export) and [warehouse docs](/docs/data-warehouse/setup) for a complete list of integrations.

<ComparisonTable column1="PostHog" column2="Google Analytics">
  <ComparisonRow column1={true} column2={true} feature="Stripe" description="Import charges, customers, invoices, subscriptions, and more" />
  <ComparisonRow column1={true} column2={true} feature="Zapier" description="Trigger Zaps with events" />

>   <ComparisonRow column1={true} column2={false} feature="Hubspot" description="Sync event and person data" />

Reviewer: ```suggestion
  <ComparisonRow column1={true} column2={false} feature="Hubspot" description="Import contacts, companies, deals, and more" />
```

>   <ComparisonRow column1={false} column2={true} feature="Google Ads" description="Track Google ad ROI" />

Reviewer: ```suggestion
  <ComparisonRow column1={false} column2={true} feature="Google Ads" description="Track Google Ads ROI" />
```
